### PR TITLE
fix(@angular/cli): show rxjs and fix alignment

### LINKED ISSUE #10257: ng version should report rxjs versions
### Versions
```
Angular CLI: 6.0.0-rc.2
Node: 9.2.0
OS: linux x64
Angular: 6.0.0-rc.3
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router
... service-worker

@angular/cdk: 6.0.0-rc.1
@angular/cli: 6.0.0-rc.2
@angular/material: 6.0.0-rc.1
@angular-devkit/architect: 0.5.4
@angular-devkit/build-angular: 0.5.4
@angular-devkit/build-optimizer: 0.5.4
@angular-devkit/core: 0.5.4
@angular-devkit/schematics: 0.5.4
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 6.0.0-rc.2.4
@schematics/angular: 0.5.4
@schematics/update: 0.5.4
typescript: 2.7.2
webpack: 4.5.0
```

### Repro steps
`ng version`

### Observed behavior
RxJS version is not printed

### Desired behavior
show me my version of rxjs and rxjs-compat (if installed)

### PR DESCRIPTION
**Pull request by hansl**
Refactored a lot of this too so its much easier to do in the future as well.

Fix #10257

### CODE REVIEW

**Comment by hansl** [comment]
image

File: packages/@angular/cli/commands/version.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { Command, Option } from '../models/command';
-import { tags } from '@angular-devkit/core';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as child_process from 'child_process';
 import chalk from 'chalk';
```
Comment (clydin) [inline packages/@angular/cli/commands/version.ts]: Might as well take out chalk as well while cleaning up

Comment (hansl) [reply packages/@angular/cli/commands/version.ts]: Done

File: packages/@angular/cli/commands/version.ts [hunk 2]
```diff
@@ -41,87 +81,81 @@ export default class VersionCommand extends Command {
...
+    moduleName: string,
+    projectNodeModules: string,
+    cliNodeModules: string,
+  ): string {
+    try {
+      if (projectNodeModules) {
+        const modulePkg = require(path.resolve(projectNodeModules, moduleName, 'package.json'));
+        return modulePkg.version;
+      }
+    } catch (e) {
```
[the remark this answers is not in the document]
Comment (hansl) [reply packages/@angular/cli/commands/version.ts]: We're still using an old version of typescript that requires it (2.6).

**Comment by filipesilva** [comment]
What does the `(cli)` next to some items mean? I guess it means the version CLI is using but I don't get why `@angular-devkit/architect` doesn't have it.

**Comment by hansl** [comment]
@filipesilva it means that dependency was not found in `node_modules/`, but `node_modules/@angular/cli/node_modules`. I'm not sure how to express this.

**Pull request merged**